Classes/Utility/FLEXColor.h

```diff
@@ -0,0 +1,26 @@
+//
+//  FLEXColor.h
+//  FLEX
+//
+//  Created by Benny Wong on 6/18/19.
+//  Copyright © 2019 Flipboard. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FLEXColor : NSObject
+
++ (UIColor *)backgoundColor;
++ (UIColor *)backgoundColorWithAlpha:(CGFloat)alpha;
++ (UIColor *)secondaryBackgroundColorWithAlpha:(CGFloat)alpha;
++ (UIColor *)scrollViewBackgroundColor;
+
++ (UIColor *)primaryTextColor;
```

Comment:
`textColor` and `secondaryTextColor` might be nice to mirror the background color methods